src/hostname/hostnamed.c

```diff
@@ -733,11 +733,14 @@ int main(int argc, char *argv[]) {
 
         r = sd_event_add_signal(event, NULL, SIGINT, NULL, NULL);
         if (r < 0)
-                return r;
+                log_error_errno(r, "Failed to install SIGINT handler: %m");
+                goto finish;
```

Comment:
Oh, LGTM analysis is correct. Missing `{}`.

Reply:
ouch!